program_options/program_options_parser.cpp

```diff
@@ -92,7 +92,16 @@ ProgramOptionsParser::ProgramOptionsParser()
   physics.add_options()(
       "equation_type",
       po::value<std::string>(&Options::get().equation_type_)->default_value("cahn-hilliard"),
-      "Governing equations to solve.");
+      "Governing equations to solve.")(
+      "m",
+      po::value<double>(&Options::get().ch_m_)->default_value(0.0),
```

Comment:
Run clang-format -i program_options_parser.cpp to make the formatting consistent